src/icu/break_iterator.cr

```diff
@@ -4,7 +4,7 @@ class ICU::BreakIterator
   DONE = -1
 
   @ubrk : LibICU::UBreakIterator
-  @utext : LibICU::UText*
+  @chars : Array(UInt16)
```

Comment:
Can you create an alias to this type (something like `UChars`) in the `ICU` module so it will be possible to use this definition in other classes ?